src/entities/addLiquidity/types.ts

```diff
@@ -97,6 +82,7 @@ export type AddLiquidityConfig = {
     customAddLiquidityTypes: Record<string, AddLiquidityBase>;
 };
 
+// type consumed by FE team, so should be exported here
```

Comment:
Do we know what they use it for? Is it SDK related?

Reply:
Hey @groninge01, this was a request from you, right?! Can you please help us remind why that's needed?

Reply:
From groninge:
> i think it was because i'd like to batch a joinpool with staking just like we do in our current beets frontend, we encode all data in the frontend and use the type from the 'old/current' sdk
so i asked if the new sdk would do the encoding for us and you said that wasn't a priority then, but instead you could add the type again so we can do the encoding in the frontend (again)

Reply:
No big deal but seems like if most of the functionality is already in the FE then the type could also sit locally. Just feels funny sitting here.

Reply:
Yeah, I see your point. And I agree it's not a big deal, so if FE team prefer it this way, I'm ok on leaving it there.
I added a more descriptive comment so we don't forget the reason in a few weeks/months again 😅